R/epi_recipe.R

```diff
@@ -288,6 +289,7 @@ is_epi_recipe <- function(x) {
 #'
 add_epi_recipe <- function(
     x, recipe, ..., blueprint = default_epi_recipe_blueprint()) {
+  rlang::check_dots_empty()
```

Comment:
This one is fine because `add_recipe()` [checks](https://github.com/tidymodels/workflows/blob/b841ff0becf9a4fab4e52ac89e628ca07d015735/R/pre-action-recipe.R#L54)